app/styles/auk/_auk-additions.scss

```diff
@@ -160,3 +160,104 @@ table tr.lt-expanded-row {
     fill: currentColor;
   }
 }
+
+$vl-picton-blue: #16465B;
+$vl-gray-70: #687483;
+$vl-gray-50: #8695A8;
```

Comment:
Css additions, is this temporary so we can change all the views and then switch the css from the package?

Also, you only define 3 colors but end up using a bunch more hex code colors in this file.

Reply:
Honestly just copy-pasted these from the usertest branch. Haven't looked at what is strictly necessary and what can be cleaned up yet.

Reply:
I think the cleanup of this will be something for me. I'll have to go through all the additions (these and previous ones) and tweak/move (to `au-kaleidos-css`) styling where needed. Blocking this PR won't be needed because I'm only be able to dedicate time to this in the following spint (85).